Rocket.Chat/Controllers/Auth/OAuthViewController.swift

```diff
@@ -28,6 +28,7 @@ class OAuthViewController: OAuthWebViewController {
 
     lazy var webView: WKWebView = {
         let webView = WKWebView(frame: view.bounds)
+        webView.customUserAgent = "Mozilla/5.0 (Linux; Android 4.1.1; Galaxy Nexus Build/JRO03C) AppleWebKit/535.19 (KHTML, like Gecko) Chrome/18.0.1025.166 Mobile Safari/535.19"
```

Comment:
I **think** it'll break our GitHub implementation... let's try another one (more recent?) or let's use it only for Google?